### ISSUE TITLE
Enable "HTTP response mode" for workflows

### ISSUE THREAD

**Issue by dylburger** [PipedreamHQ/roadmap#54]
Currently, when I use `$respond()` within a workflow, I have to include `pipedream_response=1` in the query string of my request URL, or add the equivalent HTTP header. 

I don't want to do this each time if my default case is to issue HTTP responses. For example, I might build a workflow that acts as an API, that will always return HTTP responses. So I want a way to enable "response mode" so that we always issue HTTP responses in the absence of the response query string / header.

**Comment by feliperoschel** [comment]
Hi that'd be an incredible improvement for us as our biggest partner is very restrictive regarding their api calls changes.

**Comment by kstenbeck** [comment]
Yes please do this! This will make a kick ass platform even better :-)

**Comment by dylburger** [comment]
I'm very happy to announce that we just shipped this!

If you use `$respond()` anywhere in a workflow, we'll automatically issue your HTTP response when you make an HTTP request. There's no longer a need to pass the `pipedream_response=1` query string parameter or HTTP header. 

You also don't have to enable "response mode" or do anything for this to take effect. If you're using `$respond()` in your workflow, it should just work.

We've updated the [HTTP Response docs](https://docs.pipedream.com/workflows/steps/triggers/#http-responses) accordingly.

Let us know if y'all see any issues on this or have any other related feedback
